protocols/relay/tests/lib.rs

```diff
@@ -1332,138 +1326,7 @@ fn build_keep_alive_only_swarm() -> Swarm<KeepAliveBehaviour> {
         .multiplex(libp2p_yamux::YamuxConfig::default())
         .boxed();
 
-    Swarm::new(transport, KeepAliveBehaviour::default(), local_peer_id)
-}
-
-#[derive(Clone)]
-pub struct KeepAliveBehaviour {
```

Comment:
Thank you @thomaseizinger. Past-me should have done more research before writing this :innocent:

Reply:
I think a libp2p-test crate is desperately needed :D